ovirt-ansible-engine-setup.spec.in

```diff
@@ -0,0 +1,42 @@
+%global rolename engine-setup
+%global roleprefix oVirt.
+%global ansible_roles_dir ansible/roles
+
+Name: @PACKAGE_NAME@
+Summary: Ansible role to install required packages for oVirt Engine deployment, generate answerfile and run engine-setup with it.
+Version: @RPM_VERSION@
+Release: @RPM_RELEASE@%{?release_suffix}%{?dist}
+Source0: http://resources.ovirt.org/pub/src/@PACKAGE_NAME@/@PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz
+License: ASL 2.0
+Group:          Virtualization/Management
+BuildArch:      noarch
+Url:            http://www.ovirt.org
+
+Requires: ansible >= 2.4
```

Comment:
We need to add dependency to oVirt.respositories role:

`Requires: ovirt-ansible-repositories`

Reply:
Do we really want this package is **required** ? isn't it better if it's **optional** ?

Reply:
Right, let's leave it optional